lib/matplotlib/tests/test_image.py

```diff
@@ -1118,3 +1118,9 @@ def test_exact_vmin():
 
     # check than the RBGA values are the same
     assert np.all(from_image == direct_computation)
+
+
+@pytest.mark.network
+@pytest.mark.flaky
```

Comment:
We could run `python -m http.server` in the background to get an always-available server that doesn't need access to the external network, but not sure it's worth the added complexity.